faqs.rst

```diff
@@ -197,3 +197,32 @@ A special `job shell plugin <https://github.com/flux-framework/flux-pmix>`_,
 offered as a separate package, is required to bootstrap the upcoming openmpi
 5.0.x releases.  Once installed, the plugin is activated by submitting a job
 with the ``-ompi=openmpi@5`` option.
+
+---------------------------------------------------
+Why does the ``flux mini bulksubmit`` command hang?
+---------------------------------------------------
+
+The ``flux mini bulksubmit`` command works similar to GNU parallel or
+``xargs`` and is likely blocked waiting for input from ``stdin``.
+Typical usage is to send output of some command to ``bulksubmit`` and,
+like ``xargs -I``, substitute the input with ``{}``. For example:
+
+.. code-block:: console
+
+ $ seq 1 4 | flux mini submit echo {}
+
```

Comment:
suggestion, also say what this does or show the output result.

Reply:
ha, I even see a typo (`submit` instead of `bulksubmit`), so thanks for calling this out. Maybe we should use a more useful example as well

Reply:
Thanks! I added the full output (including the `--watch` option so that output is copied to the terminal). I couldn't think of a better example suitable for a FAQ, so I left the command as is.